fulcrumapp/fulcrum-core
# Add field effects

# What?

Apparently expressions-engine in fulcrum-components receives the form as a fulcrum-core Form object.
Rather than trying to make expressions work with a raw form JSON object, we're just adding field_effects support to the fulcrum-core Form

# Why?

So field_effects show up in `JSON.stringify(form)` that gets passed to expressions

# Testing

Setup a dummy project:
- `npm init`
- `yarn add fulcrum-core@github:fulcrumapp/fulcrum-core#add-field-effects`
- Make an index.js file:
```
const { Form } = require('fulcrum-core');
const form = new Form({
  elements: [],
  field_effects: { effects: [] }
});
console.log(JSON.stringify(form));
```
- Run `node index.js`
- Verify that field_effects are set properly in the output
